src/containers/error-boundary.jsx

```diff
@@ -57,15 +58,25 @@ class ErrorBoundary extends React.Component {
             if (supportedBrowser()) {
                 return <CrashMessageComponent onReload={this.handleReload} />;
             }
-            return <BrowserModalComponent onBack={this.handleBack} />;
+            return (<BrowserModalComponent
+                isRtl={this.props.isRtl}
+                onBack={this.handleBack}
+            />);
         }
         return this.props.children;
     }
 }
 
 ErrorBoundary.propTypes = {
     action: PropTypes.string.isRequired, // Used for defining tracking action
-    children: PropTypes.node
+    children: PropTypes.node,
+    isRtl: PropTypes.bool
 };
 
-export default ErrorBoundary;
+const mapStateToProps = state => ({
+    isRtl: state.locales.isRtl
+});
+
+export default connect(
+    mapStateToProps
```

Comment:
Does there need to be an empty `mapDispatchToProps` here?

Reply:
Hmmm, it's an optional parameter so I didn't think it needed it. However, re-reading the docs it says that it injects `dispatch` into your props if you don't provide one. Looks like we should provide an empty one if we don't want the `dispatch prop` warning.

Reply:
Or maybe it's fine... backpack container doesn't define `mapDispatchToProps`: https://github.com/LLK/scratch-gui/blob/6a9bb4089c3d965c99f9b46e7b830f15ce750276/src/containers/backpack.jsx#L156